src/lib.rs

```diff
@@ -1214,6 +1228,160 @@ impl Node {
 		)
 	}
 
+	/// Add funds from the on-chain wallet into an existing channel.
+	///
+	/// This provides for increasing a channel's outbound liquidity without re-balancing or closing
+	/// it. Once negotiation with the counterparty is complete, the channel remains operational
+	/// while waiting for a new funding transaction to confirm.
+	pub fn splice_in(
```

Comment:
one issue I am having with this is that there is no way to see how much we spend on fees doing this. Would be nice if this function returned it or if it was somehow included in the `PaymentDetails`

Reply:
@tnull How should we expose this? Or should we wait until the payment API changes are addressed?

Reply:
https://github.com/lightningdevkit/rust-lightning/pull/4234